deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/conf/layers/LocalResponseNormalization.java

```diff
@@ -90,6 +93,20 @@ public ParamInitializer initializer() {
         return EmptyParamInitializer.getInstance();
     }
 
+    @Override
+    public SDVariable defineLayer(@NonNull SameDiff sameDiff, @NonNull SDVariable layerInput,
+            @NonNull Map<String, SDVariable> paramTable, SDVariable mask) {
+        if(dataFormat != CNN2DFormat.NCHW)
+            throw new UnsupportedOperationException("Can't convert non-NCHW LocalResponseNormalization to SameDiff");
```

Comment:
It'll cost us some performance, but we can permute on both ends (input and output) for the NHWC case